### PR TITLE
Updating core data

### PR DESCRIPTION
**Pull request by MichaelSelsky**
Added Sender and receiver UUID 
Added CRKUser to Core Data

### CODE REVIEW

**Comment by MichaelSelsky** [comment]
@grantjbutler merged

File: Cork/Classes/Model/Core Data/CRKUser.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+//  Created by MichaelSelsky on 04/18/15.
+//  Copyright (c) 2015 Grant Butler. All rights reserved.
+//
+
+#import "_CRKUser.h"
+
+@interface CRKUser : _CRKUser
+
++ (NSUUID *)devicesUserID;
```
Comment (grantjbutler) [inline Cork/Classes/Model/Core Data/CRKUser.h]: Maybe `currentUserUUID`?

File: Cork/Classes/Model/Core Data/Generated/_CRKMessage.h [hunk 2]
```diff
@@ -14,6 +14,8 @@
 
 @property (nonatomic, copy) NSString* messageID;
 @property (nonatomic, copy) NSString* messageText;
+@property (nonatomic, copy) NSUUID* recipientID;
+@property (nonatomic, copy) NSUUID* senderID;
```
Comment (grantjbutler) [inline Cork/Classes/Model/Core Data/Generated/_CRKMessage.h]: Since we have a `CRKUser` entity now, these should be a relationship.

**Comment by MichaelSelsky** [comment]
Didn't even know about that. #Noob

**Pull request merged**